src/components/workList.js

```diff
@@ -3,10 +3,11 @@ import WorkEntry from './workEntry'
 import data from '../data/workHistory.json'
 
 function workList() {
-    const spaces = data.map(({ company, title, image, started, ended, description }) => {
+    const spaces = data.map((item) => {
         return (
           <WorkEntry
-            entry={data}
+            entry={item}
+            key={item.company}
```

Comment:
React asks that every component created from a map have a unique value. I used `item.company` but I know that's not sustainable. You could do anything here, just as long as two items do not have the same key.